packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/nodeexpress/whatyouneed.md

```diff
@@ -0,0 +1,3 @@
+* [Set up your Okta org for a password optional use case](/docs/guides/oie-embedded-common-org-setup/android/main/#set-up-your-okta-org-for-a-password-optional-use-case)
```

Comment:
SInce, there is additional setup that needs to happen, I wanted to remind the reader that they need to set up their org for password optional. I could remove it and not mention it but it feels like something is missing and the reader might miss it.

Reply:
I removed it from What you need since the other previous guides did not place it there as well.